src/query/select.rs

```diff
@@ -283,6 +285,22 @@ impl SelectStatement {
         self
     }
 
+    /// Select distinct on
```

Comment:
Hey @oonxt, all in all the PR looks good to me. Just a nitpick, can we add some example here? Also, should we emphasize on the docs that the `distinct_on` method only works for PostgreSQL?

Reply:
Sorry for late. I'll make a new pr later.